rebar.lock

```diff
@@ -1,26 +1,36 @@
 {"1.1.0",
-[{<<"cowlib">>,{pkg,<<"cowlib">>,<<"2.3.0">>},0},
+[{<<"aleppo">>,{pkg,<<"inaka_aleppo">>,<<"1.1.1">>},2},
+ {<<"cowlib">>,{pkg,<<"cowlib">>,<<"2.3.0">>},0},
  {<<"docsh">>,{pkg,<<"docsh">>,<<"0.7.2">>},0},
+ {<<"elvis">>,{pkg,<<"elvis_core">>,<<"0.4.2">>},0},
+ {<<"elvis_core">>,{pkg,<<"elvis_core">>,<<"0.4.2">>},0},
  {<<"getopt">>,{pkg,<<"getopt">>,<<"1.0.1">>},2},
  {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
  {<<"jsx">>,{pkg,<<"jsx">>,<<"2.9.0">>},0},
+ {<<"katana_code">>,{pkg,<<"katana_code">>,<<"0.1.2">>},1},
  {<<"lager">>,{pkg,<<"lager">>,<<"3.6.8">>},0},
  {<<"providers">>,{pkg,<<"providers">>,<<"1.8.1">>},1},
  {<<"ranch">>,{pkg,<<"ranch">>,<<"1.7.1">>},0},
  {<<"redbug">>,{pkg,<<"redbug">>,<<"1.2.1">>},0},
  {<<"worker_pool">>,{pkg,<<"worker_pool">>,<<"4.0.1">>},0},
- {<<"yamerl">>,{pkg,<<"yamerl">>,<<"0.7.0">>},0}]}.
+ {<<"yamerl">>,{pkg,<<"yamerl">>,<<"0.7.0">>},0},
+ {<<"zipper">>,{pkg,<<"zipper">>,<<"1.0.1">>},1}]}.
 [
 {pkg_hash,[
+ {<<"aleppo">>, <<"B0F7D05A118C5EBC3A735D68E92A7BD5837FE53455495EDE37CF0AA8BA4912E3">>},
  {<<"cowlib">>, <<"BBD58EF537904E4F7C1DD62E6AA8BC831C8183CE4EFA9BD1150164FE15BE4CAA">>},
  {<<"docsh">>, <<"F893D5317A0E14269DD7FE79CF95FB6B9BA23513DA0480EC6E77C73221CAE4F2">>},
+ {<<"elvis">>, <<"ED5D861A38493B7750427DAECFE0D844001E017C9DBC1F7F8DE302DB6226BC24">>},
```

Comment:
And this one I guess?

Reply:
I get

```
===> Dialyzer starting, this may take a while...
===> Updating plt...
===> Could not find application: elvis
```
If I take it out.   Which probably means it needs to go back in the `rebar.config` too?